google/cloud/pubsub/internal/batch_callback_wrapper.h

```diff
@@ -55,6 +55,8 @@ class BatchCallbackWrapper : public BatchCallback {
     child_->EndMessage(ack_id, event);
   };
 
+  void AddEvent(std::string const&, std::string const&) override {};
+
```

Comment:
Are we missing `private` access control here?

Reply:
Yup